### PR TITLE
fix(curriculum): Convert blockquote elements to triple backtick syntax for Apis And Microservices

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

**PR Purpose:** Converts `blockquote` elements to triple backtick syntax in the `Description` and `Instructions` sections of the `Apis And Microservices` challenges.

Related to issue #35991

### CODE REVIEW

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-a-model.english.md [hunk 1]
```diff
@@ -14,26 +14,31 @@ A model allows you to create instances of your objects, called documents.
...
 ## Instructions
 <section id='instructions'>
 Create a person having this prototype :
-<blockquote>
-- Person Prototype -<br>
---------------------<br>
-name : string [required]<br>
-age :  number<br>
+
+```js
```
Comment (moT01) [inline curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-a-model.english.md]: This probably doesn't qualify as code. Revert to blockquotes or unlabeled backticks?

**Review by moT01 — COMMENTED** [review]
There's one PR ahead of this https://github.com/freeCodeCamp/freeCodeCamp/pull/35925

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-a-model.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-a-model.english.md]: I reverted it to blockquote.

**Comment by RandellDawson** [comment]
@moT01 I resolved the merge conflict on this one.

File: curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-data-from-post-requests.english.md [hunk 2]
```diff
@@ -7,7 +7,13 @@ challengeType: 2
 ## Description
 <section id='description'>
 Mount a POST handler at the path <code>/name</code>. It’s the same path as before. We have prepared a form in the html frontpage. It will submit the same data of exercise 10 (Query string). If the body-parser is configured correctly, you should find the parameters in the object <code>req.body</code>. Have a look at the usual library example:
-<blockquote>route: POST '/library'<br>urlencoded_body: userId=546&bookId=6754 <br>req.body: {userId: '546', bookId: '6754'}</blockquote>
+
+```js
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-data-from-post-requests.english.md]: I'm not sure about this one - it doesn't look like it's supposed to be code.  Well, the last line `req.body: {userId: '546', bookId: '6754'}` is, but not the rest.  Maybe leave it as a blockquote.

File: curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-query-parameter-input-from-the-client.english.md [hunk 3]
```diff
@@ -7,7 +7,13 @@ challengeType: 2
 ## Description
 <section id='description'>
 Another common way to get input from the client is by encoding the data after the route path, using a query string. The query string is delimited by a question mark (?), and includes field=value couples. Each couple is separated by an ampersand (&). Express can parse the data from the query string, and populate the object <code>req.query</code>. Some characters, like the percent (%), cannot be in URLs and have to be encoded in a different format before you can send them. If you use the API from JavaScript, you can use specific methods to encode/decode these characters.
-<blockquote>route_path: '/library'<br>actual_request_URL: '/library?userId=546&bookId=6754' <br>req.query: {userId: '546', bookId: '6754'}</blockquote>
+
+```js
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-query-parameter-input-from-the-client.english.md]: Same problem as above.

File: curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-route-parameter-input-from-the-client.english.md [hunk 4]
```diff
@@ -7,7 +7,13 @@ challengeType: 2
 ## Description
 <section id='description'>
 When building an API, we have to allow users to communicate to us what they want to get from our service. For example, if the client is requesting information about a user stored in the database, they need a way to let us know which user they're interested in. One possible way to achieve this result is by using route parameters. Route parameters are named segments of the URL, delimited by slashes (/). Each segment captures the value of the part of the URL which matches its position. The captured values can be found in the <code>req.params</code> object.
-<blockquote>route_path: '/user/:userId/book/:bookId'<br>actual_request_URL: '/user/546/book/6754' <br>req.params: {userId: '546', bookId: '6754'}</blockquote>
+
+```js
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-route-parameter-input-from-the-client.english.md]: And again.

File: curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/use-body-parser-to-parse-post-requests.english.md [hunk 5]
```diff
@@ -8,7 +8,16 @@ challengeType: 2
 <section id='description'>
 Besides GET, there is another common HTTP verb, it is POST. POST is the default method used to send client data with HTML forms. In REST convention, POST is used to send data to create new items in the database (a new user, or a new blog post). You don’t have a database in this project, but you are going to learn how to handle POST requests anyway.
 In these kind of requests, the data doesn’t appear in the URL, it is hidden in the request body. This is a part of the HTML request, also called payload. Since HTML is text-based, even if you don’t see the data, it doesn’t mean that it is secret. The raw content of an HTTP POST request is shown below:
-<blockquote>POST /path/subpath HTTP/1.0<br>From: john@example.com<br>User-Agent: someBrowser/1.0<br>Content-Type: application/x-www-form-urlencoded<br>Content-Length: 20<br>name=John+Doe&age=25</blockquote>
+
+```js
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/use-body-parser-to-parse-post-requests.english.md]:
```suggestion
```http
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
I'm not sure what the correct approach is for those slightly odd blocks, but I lean towards leaving them as blockquotes.

File: curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-data-from-post-requests.english.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-data-from-post-requests.english.md]: I reverted this one back to blockquote just now.

File: curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-query-parameter-input-from-the-client.english.md (hunk 3 above)
Comment (RandellDawson) [reply curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-query-parameter-input-from-the-client.english.md]: I have reverted this one back to blockquote now.

File: curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-route-parameter-input-from-the-client.english.md (hunk 4 above)
Comment (RandellDawson) [reply curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/get-route-parameter-input-from-the-client.english.md]: I have reverted this one back to blockquote.

**Comment by RandellDawson** [comment]
@ojeytonwilliams I think I have resolved the issues on this PR.

**Review by moT01 — APPROVED** [review]
Tested again locally - LGTM 🎉

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
Thanks for making the changes, LGTM :+1:

**Comment by RandellDawson** [comment]
@moT01 @ojeytonwilliams See my comment on #35993. I believe this can be merged now.

**Comment by moT01** [comment]
🎉

**Pull request merged**